packages/flutter/lib/src/cupertino/magnifier.dart

```diff
@@ -161,17 +152,12 @@ class _CupertinoTextMagnifierState extends State<CupertinoTextMagnifier>
     final double verticalPositionOfLens = math.max(
         verticalCenterOfCurrentLine,
         verticalCenterOfCurrentLine -
-            (verticalCenterOfCurrentLine -
-                    textEditingContext.globalGesturePosition.dy) /
-                widget.dragResistance);
+            (verticalCenterOfCurrentLine - textEditingContext.globalGesturePosition.dy) / widget.dragResistance);
 
     // The raw position, tracking the gesture directly.
```

Comment:
Gere is a case where we don't really seem to follow our own rules about expression formatting and so it kind of looks bad on either side